files/id-stg/docker-compose.yml

```diff
@@ -1,40 +1,40 @@
-version: "3"
+version: '3'
+
 services:
-  mongodb:
-    restart: "always"
-    image: mongo:3.4.9
-    healthcheck:
-      test: ["CMD", "mongo", "--quiet", "localhost/test", "--eval", "'quit(db.runCommand({ ping: 1 }).ok ? 0 : 1)';"]
-      interval: 1m30s
-      timeout: 10s
-      retries: 3
+  postgres:
+    image: postgres
     volumes:
-      - mongo-data:/data/db
-      - mongo-backup:/opt/backup
-
-  id-web:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB:-keycloak}
+      POSTGRES_USER: ${POSTGRES_USER:-keycloak}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-password}
     restart: "always"
-    image: openmrsinfra/id:staging
-    ports:
-      - "127.0.0.1:3000:3000"
-    env_file: .env
+  keycloak:
+    image: openmrs/openmrs-id-keycloak:latest
     environment:
-      NODE_ENV: "production"
-      NODE_TLS_REJECT_UNAUTHORIZED: "0"
+      - KC_HOSTNAME=${HOSTNAME:-localhost}
+      - KC_DB=postgres
+      - KC_DB_URL_DATABASE=${POSTGRES_DB:-keycloak}
+      - KC_DB_URL_HOST=postgres
+      - KC_DB_URL_PORT=5432
+      - KC_DB_USERNAME=${POSTGRES_USER:-keycloak}
+      - KC_DB_PASSWORD=${POSTGRES_PASSWORD:-password}
+      - KEYCLOAK_ADMIN=${KEYCLOAK_ADMIN:-admin}
+      - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD:-password}
+      - KC_PROXY:${KC_PROXY:-none}
+      - PROXY_ADDRESS_FORWARDING=${PROXY_ADDRESS_FORWARDING:-false}
+    restart: "always"
+    ports:
+      - "8443:8443"
```

Comment:
Since we'll be handling ssl via nginx we need to expose http port for Keycloak and not https. Change it to 8080, which is Keycloak's default.